props/saber_fett263_buttons.h

```diff
@@ -381,160 +1813,2794 @@ SaberFett263Buttons() : PropBase() {}
     if (dynamic_mixer.get_volume() < VOLUME) {
       dynamic_mixer.set_volume(std::min<int>(VOLUME + VOLUME * 0.1,
                                              dynamic_mixer.get_volume() + VOLUME * 0.10));
-      beeper.Beep(0.5, 2000);
+      PlayMenuSound("volup.wav");
       STDOUT.print("Current Volume: ");
       STDOUT.println(dynamic_mixer.get_volume());
     } else {
-      beeper.Beep(0.5, 3000);
+      PlayMenuSound("volmax.wav");
     }
+    return;
   }
 
   void VolumeDown() {
     STDOUT.println("Volume Down");
     if (dynamic_mixer.get_volume() > (0.10 * VOLUME)) {
       dynamic_mixer.set_volume(std::max<int>(VOLUME * 0.1,
                                              dynamic_mixer.get_volume() - VOLUME * 0.10));
-      beeper.Beep(0.5, 2000);
+      PlayMenuSound("voldown.wav");
       STDOUT.print("Current Volume: ");
       STDOUT.println(dynamic_mixer.get_volume());
     } else {
-      beeper.Beep(0.5, 1000);
+      PlayMenuSound("volmin.wav");
     }
+    return;
   }
 
+RefPtr<BufferedWavPlayer> wav_player;
+
+// Menu Sound Player
+void PlayMenuSound(const char* file) {
```

Comment:
Would it be better to put it on the queue?

Reply:
I don't think so, the menu "waits" for each individual prompt to complete before recognizing next dial twist.  This is so larger twist doesn't jump ahead before the sound.  If we let the dial recognize a new value before the prompt finishes playing or the new prompt plays then users can end up in different selection than intended so only being able to play one sound, particularly with the submenu choices, is necessary and intentional.